habitat-lab/habitat/config/default.py

```diff
@@ -16,36 +17,51 @@
     HabitatConfigPlugin,
     register_hydra_plugin,
 )
+from habitat.config.read_write import read_write
 
 _HABITAT_CFG_DIR = osp.dirname(inspect.getabsfile(inspect.currentframe()))
 # This is equivalent to doing osp.dirname(osp.abspath(__file__))
 # in editable install, this is pwd/habitat-lab/habitat/config
 CONFIG_FILE_SEPARATOR = ","
 
 
-def get_full_config_path(
-    config_path: str, default_configs_dir: str = _HABITAT_CFG_DIR
-) -> str:
+def get_full_config_path(config_path: str, configs_dir: str) -> str:
     if osp.exists(config_path):
         return osp.abspath(config_path)
 
-    proposed_full_path = osp.join(default_configs_dir, config_path)
+    proposed_full_path = osp.join(configs_dir, config_path)
     if osp.exists(proposed_full_path):
         return osp.abspath(proposed_full_path)
 
     raise RuntimeError(f"No file found for config '{config_path}'")
 
 
+get_full_habitat_config_path = partial(
```

Comment:
Same here. This functions will be frequently used and should have better documentation.